### PR TITLE
Update the logo in the header section

### PR DESCRIPTION
**Pull request by SohamRoy-01**
## Fixes Issue

This PR fixes the following issues:

closes #issue-number

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes

### Before :-
Screenshot (14)

### After :-
Screenshot (23)

### CODE REVIEW

**Comment by SohamRoy-01** [comment]
@Chandraprakash-Darji Due to merge conflicts I close the PR and I removed the small logo and update it accordingly...

**Comment by Chandraprakash-Darji** [comment]
@SohamRoy-01 When you create the PR, just make share the screenshot

**Comment by SohamRoy-01** [comment]
I added the screenshot @Chandraprakash-Darji

**Review by siddhant-khisty — APPROVED** [review]
LGTM.
cc @AnuragThePathak

**Comment by AnuragThePathak** [comment]
@SohamRoy-01 kindly fix the merge conflicts and we are good to merge it.

**Comment by SohamRoy-01** [comment]
Fixed @AnuragThePathak

File: README.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-<img src="https://user-images.githubusercontent.com/114678694/208736801-03fbace3-5b00-42b9-8dfe-db18dcc13714.png" height="180" >
+  <img src="https://user-images.githubusercontent.com/114678694/208736801-03fbace3-5b00-42b9-8dfe-db18dcc13714.png" height="250">
 </p>
 
-<h1  align= "center" id="logo"><a><img src="https://user-images.githubusercontent.com/114678694/208736801-03fbace3-5b00-42b9-8dfe-db18dcc13714.png" height="40" style="vertical-align:middle;padding:0px 8px 10px 0px" alt="logo" />We Make Devs</a></h1>
+<p align="center">We Make Devs - Empowering developers to achieve their potential.</p>
 
-###
+<p align="center">
+  <a href="https://github.com/commclassroom/commclassroom/issues"><img alt="GitHub issues" src="https://img.shields.io/github/issues/commclassroom/commclassroom"></a>
```
Comment (Chandraprakash-Darji) [inline README.md:8]: This link should be of wemakedevs not the commclassrom

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SohamRoy-01) [reply README.md]: changed done,

**Review by Chandraprakash-Darji — APPROVED** [review]

**Pull request merged**